tests/helpers/Messages/createMessage.test.js

```diff
@@ -0,0 +1,32 @@
+const createMessage = require("../../../helpers/Messages/createMessage");
+const createUser = require("../../../helpers/Users/createUser");
+const { createTables, wipeDBTables } = require("../../../db/databaseHelpers");
+
+beforeAll(async() =>{
+  await createTables();
+  await createUser("1", "hel", "password", "student");
+  await createUser("2", "jka", "pasd", "tutor");
+});
+afterAll(async() =>{
+  await wipeDBTables();
+});
+describe("creating a message", () =>{
+  it("should return true", async() =>{
+    const message = await createMessage("99", "1", "2", "hello");
+    expect(message).toEqual(true);
+  });
+});
+describe("message id already exists", () =>{
+  it("should return false", async() =>{
+    await createMessage("99", "1", "2", "hello");
```

Comment:
since the ID's are strings, use friendly test id so it's easy to read: `test_msg_id` and `student_id` etc...

Reply:
8c3aeb3b44554337da4866182ce3d112cc5485be